index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello Microverse</h1>
```

Comment:
- [ ] It is required to give a class name to your `h1` element and use that class name inside your style.css to style the content of the `h1` tag.

![style](https://user-images.githubusercontent.com/26597432/235257374-98c3bb9f-4bd3-4ca7-a881-75a59a881e3c.png)